checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakVisitor.java

```diff
@@ -312,6 +313,35 @@ private static String getCreatesMustCallForValue(
     return result;
   }
 
+  /**
+   * Get all {@link EnsuresCalledMethods} annotations on an element.
+   *
+   * @param elt an executable element that might have {@link EnsuresCalledMethods} annotations
+   * @param atypeFactory a <code>ResourceLeakAnnotatedTypeFactory</code>
+   * @return a set of {@link EnsuresCalledMethods} annotations
+   */
+  @Pure
+  private static AnnotationMirrorSet getEnsuresCalledMethodsAnnotations(
+      ExecutableElement elt, ResourceLeakAnnotatedTypeFactory atypeFactory) {
+    AnnotationMirror createsMustCallForList =
```

Comment:
I assume that this code is based on `CreatesMustCallForToJavaExpression#getCreatesMustCallForAnnos`? I think that is the right template (`@CreatesMustCallFor` is also a repeatable declaration annotation), but names should be changed to reflect the new context: this variable should be `ensuresCalledMethodsList`, `createsMustCallFors` should be `ensuresCalledMethodsAnnos` or similar, etc.